com.unity.render-pipelines.universal/Shaders/Utils/Deferred.hlsl

```diff
@@ -7,7 +7,7 @@
 
 #define PREFERRED_CBUFFER_SIZE (64 * 1024)
 #define SIZEOF_VEC4_TILEDATA 1 // uint4
-#define SIZEOF_VEC4_PUNCTUALLIGHTDATA 5 // 5 * float4
+#define SIZEOF_VEC4_PUNCTUALLIGHTDATA 6 // 6 * float4
```

Comment:
Maybe this could be auto generated from C# as well.

Reply:
I remember HDRP has something for that, do we also have something similar in URP?

Reply:
Actually found the tool. The "size" of the structure in float4 units is needed when manually trying to read back structure from an uniform buffer (no officially supported in Unity). But the tool will not generate actual "size" of the structure, so I am leaving it as it is for now.

Reply:
Ok. Not blocking issue.